lms/djangoapps/course_home_api/progress/v1/tests/test_views.py

```diff
@@ -14,6 +14,8 @@
 from student.models import CourseEnrollment
 from student.tests.factories import UserFactory
 
+CREDIT_SUPPORT_URL = 'https://support.edx.org/hc/en-us/sections/115004154688-Purchasing-Academic-Credit'
+
 
 @override_waffle_flag(COURSE_HOME_MICROFRONTEND, active=True)
```

Comment:
So I also think there should be some tests regarding the credit_course_requirements and the verification data.